doc/Developing/Style-Guidelines.md

```diff
@@ -1,76 +0,0 @@
-source: Developing/Style-Guidelines.md
```

Comment:
We should maintain a style guide still but we can probably bring this back later

Reply:
Yes sure but it's outdated and not really saying much